package.json

```diff
@@ -4,14 +4,16 @@
   "description": "Setup VS dev environment for GitHub Actions",
   "main": "dist/index.js",
   "author": "Sean Middleditch",
-  "license": "CC0",
+  "license": "MIT",
```

Comment:
Please revert this; I do not want to require attribution for folks who fork/reuse code this trivial.